package.json

```diff
@@ -7,7 +7,7 @@
     "integration": "src/bin/integration"
   },
   "scripts": {
-    "test": "node ./src/bin/integration all",
+    "test": "npm install five-bells-service-manager && node ./src/bin/integration all",
```

Comment:
I change this line so that`five-bells-integration-test-loader` is used. It will install matching branch versions of `five-bells-service-manager` and `five-bells-integration-test`.